src/models/setting.model.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  BelongsTo,
+  Column,
+  Model,
+  Table,
+  DeletedAt,
+  CreatedAt,
+  UpdatedAt,
+  ForeignKey,
+  AllowNull,
+} from 'sequelize-typescript';
+import Account from 'src/models/account.model';
+
+@Table({
+  indexes: [{ fields: ['account_id', 'name'], unique: true }],
+})
+export default class Setting extends Model {
+  @AllowNull(false)
+  @Column
+  name: string;
+
+  @AllowNull(false)
+  @Column
+  data_type: string;
```

Comment:
This should by an enum instead of string

Reply:
[Fixed](https://github.com/ananth243/nest-starter-project/blob/732f3cf7c7bc7cf884500fc5eda0ff5aa6709c60/src/models/setting.model.ts#L29)